package.json

```diff
@@ -12,7 +12,7 @@
     "compile": "tsc",
     "build": "yarn clean; yarn compile",
     "start": "node -r module-alias/register ./dist/index.js",
-    "migrate": "node ./node_modules/node-pg-migrate/bin/node-pg-migrate -m ./src/migrations --no-check-order",
+    "migrate": "NODE_TLS_REJECT_UNAUTHORIZED=0 node ./node_modules/node-pg-migrate/bin/node-pg-migrate -m ./src/migrations --no-check-order",
```

Comment:
Why is this being done exactly? I googled this env variable and found this. Please have a look

[Node.JS Error- process.env.NODE_TLS_REJECT_UNAUTHORIZED. What does this mean?](https://stackoverflow.com/questions/35633829/node-js-error-process-env-node-tls-reject-unauthorized-what-does-this-mean)

Reply:
I haven't install ssl certificate from rds yet, will do in some time, till that till this will allow migration, this is only for migration rest all working properly in SSL mode